### PR TITLE
Prep for beta 1 release

### PR DESCRIPTION
**Pull request by patrickarlt**
This preps for the beta 1 release. Which I will run as soon as this gets merged and I have admin privileges on this repo.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -4,8 +4,8 @@ Esri welcomes contributions from anyone and everyone. Please see our [guidelines
...
+* Adding ideas for components to out [wishlist project](https://github.com/Esri/calcite-components/projects/2) We are using the wishlist to gather ideas about what components teams are using that we might be able to reproduce as a part of Calcite Components.
+* If you want to help develop components take a look at out [1.0.0 components](https://github.com/Esri/calcite-components/projects/1) which are the components we are targeting for the first release of Calcite Components. Before starting development please review our [component conventions](./conventions/README.md) and the [Stencil documentation](https://stenciljs.com/docs/host-element). We also have a basic [`<calcite-element>`](./components/calcite-example) component that can be copied as a starter point for a new component.
```
Comment (jcfranco) [inline CONTRIBUTING.md]: `out` -> `our`?

File: src/components/calcite-alert/readme.md [hunk 2]
```diff
@@ -47,7 +46,7 @@ Single alert for toast notifications. To position the alert correctly and manage
 
 ### `closeCalciteAlert() => Promise<void>`
 
-emit the `calciteAlerClose` event - <calcite-alerts> listens for this, if the alert is not active, but is the queue, this will remove it from the queue
+emit the `calciteAlerClose` event - <calcite-alerts> listens for this
```
Comment (jcfranco) [inline src/components/calcite-alert/readme.md]: `calciteAlerClose` -> `calciteAlertClose`.

File: src/components/calcite-alerts/readme.md [hunk 3]
```diff
@@ -12,19 +12,12 @@ The `calcite-alerts` component is meant to wrap one or multiple [calcite-alert](
...
-
 ## Events
 
-| Event                | Description | Type               |
-| -------------------- | ----------- | ------------------ |
-| `calciteAlertsClose` |             | `CustomEvent<any>` |
-| `calciteAlertsOpen`  |             | `CustomEvent<any>` |
+| Event                | Description                                                                         | Type               |
+| -------------------- | ----------------------------------------------------------------------------------- | ------------------ |
+| `calciteAlertsClose` | emits the id of the alert ot be closed, and the current alertQueue and currentAlert | `CustomEvent<any>` |
```
Comment (jcfranco) [inline src/components/calcite-alerts/readme.md]: `id` -> `ID`, `ot` -> `to`.

File: src/components/calcite-alert/readme.md (hunk 2 above)
Comment (macandcheese) [reply src/components/calcite-alert/readme.md]: This is mine. Will fix.

File: src/components/calcite-alerts/readme.md (hunk 3 above)
Comment (macandcheese) [reply src/components/calcite-alerts/readme.md]: This is mine. Will fix.

**Pull request merged**